miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridge.kt

```diff
@@ -100,8 +102,12 @@ internal class UserInfoBridge {
                 accessToken.scopes = tokenPermission
                 bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
             }
-            val errorCallback = { message: String ->
-                bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
+
+            // send a specifically formatted error key to show specific error in mini app
+            val errorCallback = { callback: MiniAppAccessTokenError ->
+                // convert error callback to common MiniAppBridgeErrorModel
+                val errorBridgeModel = MiniAppBridgeErrorModel(callback.type, callback.message)
+                bridgeExecutor.postError(callbackObj.id, Gson().toJson(errorBridgeModel))
             }
```

Comment:
this has to be done on line 115 where you just send a regular `String` as an error and not a `MiniAppBridgeErrorModel`

I think you will have to modify `doesAccessTokenMatch` method to return the right error type and not just a Boolean